src/Content.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useMemo } from 'react';
+import { Person } from './types/Person';
+import classNames from 'classnames';
+import { Dropdown } from './Dropdown';
+
+interface ContentProps {
+  debounce: (
+    fn: (...args: string[]) => void,
+    delay: number,
+  ) => (...args: [string]) => void;
+  delay: number;
+  peopleFromServer: Person[];
+}
+
+export const Content: React.FC<ContentProps> = ({
+  debounce,
+  delay,
+  peopleFromServer,
+}) => {
+  const [query, setQuery] = React.useState('');
+  const [appliedQuery, setAppliedQuery] = React.useState('');
+  const [isActive, setIsActive] = React.useState(false);
+  // const [people, setPeople] = React.useState(peopleFromServer);
+  const [selectedPerson, setSelectedPerson] = React.useState<Person | null>(
+    null,
+  );
+
+  const handleFocus = () => {
+    setIsActive(true);
+  };
+
+  const deboncedSetQuery = useMemo(
+    () => debounce((value: string) => setAppliedQuery(value), delay),
+    [debounce, delay],
+  );
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
```

Comment:
Consider adding a check to prevent setting the query if the input contains only spaces. This will help avoid unnecessary filtering operations.